build_calico_kubernetes/requirements.txt

```diff
@@ -1,5 +1,5 @@
 PyInstaller==2.1
-git+https://github.com/projectcalico/libcalico.git@v0.2.0
+git+https://github.com/robbrockbank/libcalico.git@policy-only
```

Comment:
There will need to be a update to libcalico with relevant changes for this PR to go in.